day2.js

```diff
@@ -0,0 +1,55 @@
+function vowelsAndConsonants(s) {
+
+
+    const vowels = 'aeiou';
+
+
+    var consonants = '';
+
+
+
+
+    for (var i = 0; i < s.length; i++) {
+
+
+        if (vowels.includes(s[i])) {
+
+
+            console.log(s[i]);
+
+
+        }
+
+
+        else {
+
+
+            consonants += s[i] + '\n';
+
+
+        }
+
+
+    }
+    console.log(consonants.trim());
+
+
+}
+
+
+
+
+
+
+function main() {
+
+
+    const s = readLine();
+
+
+
+
+    vowelsAndConsonants(s);
+
+
+}
```

Comment:
too much spaces, this is not good coding

Reply:
yes hii sikuiona mwalimu

Reply:
next time sintorudia kosa